sdks/python/apache_beam/dataflow_test.py

```diff
@@ -109,11 +109,12 @@ def test_par_do_with_side_input_as_keyword_arg(self):
 
   @attr('ValidatesRunner')
   def test_par_do_with_do_fn_object(self):
-    class SomeDoFn(DoFn):
+    class SomeDoFn(NewDoFn):
       """A custom DoFn for a FlatMap transform."""
 
-      def process(self, context, prefix, suffix):
-        return ['%s-%s-%s' % (prefix, context.element, suffix)]
+      def process(self, prefix, element=NewDoFn.ElementParam,
```

Comment:
also, NewDoFn.ElementParam should be default on first argument. Thus this signature (and most others) wouldn't change but for context -> element.

Probably makes sense to nail API before changing all examples (in a separate commit).